tests/integration/sharding_tests/test_sharding_tls.py

```diff
@@ -0,0 +1,324 @@
+#!/usr/bin/env python3
```

Comment:
nit:

I know in the VM charm this test is in the directory:
`tests/integration/sharding_tests`

but this scheme is not ideal, instead can you move this test to:
`tests/integration/tls_tests/test_sharding_tls.py`

Reply:
Sure